snap/gadget.go

```diff
@@ -69,6 +72,71 @@ type VolumeContent struct {
 	Unpack bool `yaml:"unpack"`
 }
 
+// GadgetConnect describes an interface connection requested by the gadget
+// between seeded snaps. The syntax looks like this:
+//
+// [<plug-snap-id1>|system]:plug [[<slot-snap-id>|system]:slot]
```

Comment:
This is nice, I like `system` here. Do you think this is settled or do we still need to make a distinction that `snapd` may offer implicit slots?

Reply:
"system" should be fine here.